lib/rabbitmq_message_deduplication/cache.ex

```diff
@@ -230,9 +245,25 @@ defmodule RabbitMQMessageDeduplication.Cache do
     end
   end
 
+  defp _ensure_distributed(cache) do
```

Comment:
In Elixir prefixing names with underscore is used to inform the compiler that a given variable is not expected to be used (and therefore silence unused vars warnings).

Contrary to Python, it is not used for private functions. For that, the `defp` form is used instead.

Reply:
A better name than `_distribute_cache` would be, for example, `maybe_replicate` as RMQ uses this naming convention for functions with conditional side effects.